srcpkgs/syncthing/files/syncthing/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+
+export USERNAME=
+export HOME="/home/$USERNAME"
```

Comment:
Those variables should be defined before sourcing `conf` to allow using the config file to define them without editing the `run` file which gets overwritten on updates.